tests/integration/components/bs-carousel-test.js

```diff
@@ -184,6 +192,8 @@ module('Integration | Component | bs-carousel', function(hooks) {
       hbs`{{#bs-carousel autoPlay=true interval=interval transitionDuration=transitionDuration ltr=false as |car|}}{{car.slide}}{{car.slide}}{{car.slide}}{{/bs-carousel}}`
     );
     await waitTransitionTime();
+    assert.notOk(getActivatedSlide(1), 'ltr has correct behavior');
+    assert.notOk(getActivatedSlide(1), 'ltr has correct behavior');
```

Comment:
should this be like this?

```suggestion
    assert.notOk(getActivatedSlide(2), 'ltr has correct behavior');
```

Reply:
yup! I'll get that fixed